CHANGELOG.md

```diff
@@ -0,0 +1,62 @@
+# Changelog
+All notable changes to this project will be documented in this file.
+
+The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
+and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
+
+
+## [Unreleased]
+
+### Added
+
+* Integrating APIs with security from [@ohltyler](https://github.com/ohltyler)
+
+### Changed 
+* Corrected ml/stats/nodes permissions spelling from [@camAtGitHub](https://github.com/camAtGitHub)
+* Internal user usage clarification from [@reshippie](https://github.com/reshippie)
+* Remove `advanced_modules_enabled` examples from [@wjordan](https://github.com/wjordan)
+* CHANGELOG.md replaces release-notes
+
+### Security
+
+* Security patch for `org.cryptacular:cryptacular`
+* Security patch for `org.springframework:spring-core`
+* Security patch for `org.apache.kafka:kafka-clients`
+* Security patch for `org.springframework.kafka:spring-kafka-test`
+
+## [2.0.0-rc1] - 2022-04-20
+
+### Added
+
+* DLS privileges evaluation can process security roles from [@ch-govau](https://github.com/ch-govau)
+* Setting to enable role without DLS/FLS to override roles
+* DLS Term Lookup Queries
+
+### Changed 
+
+* Support for JDK17
+* Support for Gradle 7
+* Use standard Issue/Pull request templates from [@dblock](https://github.com/dblock)
+* Security configuration are placed side-by-side with other plugins
+* Cleaned up developer guide
+
+### Deprecated
+
+* Security tools deprecation
+
+### Removed
+
+* No longer supporting JDK14
+* Removed TransportClient
```

Comment:
i'd have expected that OpenSearch has removed the `TransportClient` and the security plugin has just adapted to this? in that case this wouldn't be under `Removed` but would instead be listed under `Changed` as `Adapt to OpenSearch 2.0` (again, a user doesn't care what exactly you had to do to adapt to 2.0 - and if they do they can just follow the link to the ticket which hopefully then leads them to everything else; and worst-case they can look at the commit history and, if the commit messages were meaningful, can even filter on them because they might all contain a comment indicating that they're relevant for 2.0)

Reply:
I think this change goes into the 'inards' of the repository/test systems that are not relevant to consumers.  I'll omit this change.